stac_fastapi/types/stac_fastapi/types/search.py

```diff
@@ -170,9 +201,7 @@ class BaseSearchGetRequest(APIRequest):
             },
         ),
     ] = attr.ib(default=None)
-    datetime: Optional[DateTimeType] = attr.ib(
-        default=None, converter=_datetime_converter
-    )
+    datetime: DateTimeQueryType = attr.ib(default=None, validator=_validate_datetime)
```

Comment:
Technically we didn't had to add `datetime` attribute here, but we do it to keep the `order` of the attributes the same as before (mostly for openapi docs 🤷‍♂️)